symengine/cwrapper.cpp

```diff
@@ -737,14 +737,22 @@ void vecbasic_free(CVecBasic *self)
     delete self;
 }
 
-void vecbasic_push_back(CVecBasic *self, const basic value)
+CWRAPPER_OUTPUT_TYPE vecbasic_push_back(CVecBasic *self, const basic value)
 {
+    CWRAPPER_BEGIN
+
     self->m.push_back(value->m);
+
+    CWRAPPER_END
 }
 
-void vecbasic_get(CVecBasic *self, int n, basic result)
+CWRAPPER_OUTPUT_TYPE vecbasic_get(CVecBasic *self, int n, basic result)
 {
+    CWRAPPER_BEGIN
+
     result->m = self->m[n];
+
```

Comment:
That's true. But should the function be checking if the casting has been done properly when the function specifically asked for a `size_t`? I believe it should be the user who should be careful of those kind of errors. Besides `size_t` are better suited to hold array indices.